dapps/profile.vue/src/components/root/root.vue

```diff
@@ -34,6 +34,7 @@
                 <template slot="header">
                   <evan-profile-preview
                     class="p-4"
+                    style="width: 240px;"
```

Comment:
shouldn't this be depends at least on the size param? May cause side effects setting it hard ...

Reply:
The sidepanel is sized dynamically using flex. If the name of the `profile-preview` gets too long, `white-space: nowrap` for ellipsis text overflow prevents line breaks and it will increase the sidepanel width into horizion.